components/DeclensionForm/DeclensionForm.js

```diff
@@ -0,0 +1,152 @@
+        <h2>{base.translation}</h2>
+        <h2>
+          {query1.translation} ({query1.gender})
+        </h2>
+      </StyledHeadingWrapper>
+      <StyledForm onSubmit={handleSubmitDeclensionForm}>
+        <StyledSpecificationWrapper>
+          <label htmlFor="specification">declension type:</label>
+          <input type="text" name="specification" />
+        </StyledSpecificationWrapper>
+        <StyledDeclensionWrapper>
+          <p></p>
+          <p></p>
+          <h4>singular</h4>
+          <p></p>
+          <h4>plural</h4>
+          <p>1</p>
+          <label htmlFor="nominativeSingular"></label>
+          <input
+            type="text"
+            name="nominativeSingular"
+            placeholder="nominative"
+          />
+          <label htmlFor="nominativePlural"></label>
+          <input type="text" name="nominativePlural" placeholder="nominative" />
+          <p>2</p>
+          <label htmlFor="genitiveSingular"></label>
+          <input type="text" name="genitiveSingular" placeholder="genitive" />
+          <label htmlFor="genitivePlural"></label>
+          <input type="text" name="genitivePlural" placeholder="genitive" />
+          <p>3</p>
+          <label htmlFor="dativeSingular"></label>
+          <input type="text" name="dativeSingular" placeholder="dative" />
+          <label htmlFor="dativePlural"></label>
+          <input type="text" name="dativePlural" placeholder="dative" />
+          <p>4</p>
+          <label htmlFor="accusativeSingular"></label>
+          <input
+            type="text"
+            name="accusativeSingular"
+            placeholder="accusative"
+          />
+          <label htmlFor="accusativePlural"></label>
+          <input type="text" name="accusativePlural" placeholder="accusative" />
+          <p>5</p>
+          <label htmlFor="locativeSingular"></label>
+          <input type="text" name="locativeSingular" placeholder="locative" />
+          <label htmlFor="locativePlural"></label>
+          <input type="text" name="locativePlural" placeholder="locative" />
+          <p>6</p>
+          <label htmlFor="instrumentalSingular"></label>
+          <input
+            type="text"
+            name="instrumentalSingular"
+            placeholder="instrumental"
+          />
+          <label htmlFor="instrumentalPlural"></label>
+          <input
+            type="text"
+            name="instrumentalPlural"
+            placeholder="instrumental"
+          />
+        </StyledDeclensionWrapper>
+        <StyledDeclensionButtonWrapper>
+          <button type="submit">add</button>
+        </StyledDeclensionButtonWrapper>
+      </StyledForm>
+      <Footer path={currentWord.category} />
+    </>
+  );
+}
+
+const StyledHeadingWrapper = styled.section`
+  margin: 10px 10px 20px 10px;
+  background-color: White;
+  color: darkmagenta;
+  text-align: center;
+  padding: 7px;
+  cursor: default;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+`;
+
```

Comment:
Please try to use relative units on margin and padding. It will prevent styling problems on different displays. Try using rem or percent.